pyuvsim/simsetup.py

```diff
@@ -600,22 +624,26 @@ def parse_telescope_params(tele_params, config_path=''):
         telescope_location_latlonalt = ast.literal_eval(telconfig['telescope_location'])
         telescope_location = list(telescope_location_latlonalt)
         telescope_location[0] *= np.pi / 180.
-        telescope_location[1] *= np.pi / 180.   # Convert to radians
+        telescope_location[1] *= np.pi / 180.  # Convert to radians
         tele_params['telescope_location'] = uvutils.XYZ_from_LatLonAlt(*telescope_location)
         telescope_name = telconfig['telescope_name']
 
     else:
         # if not provided, get bare-minumum keys from tele_params
         if 'telescope_location' not in tele_params:
-            raise KeyError("If telescope_config_name not provided in `telescope` obsparam section, you must provide telescope_location")
+            raise KeyError(
+                "If telescope_config_name not provided in `telescope` obsparam section, you must "
+                "provide telescope_location")
```

Comment:
This seems like a weird solution. We already have a two line message, it just needs to be broken at a different place.

Reply:
Sorry how is it weird?

Reply:
I think I read the diff wrong. It's ok.